cmd/fleetctl/scripts.go

```diff
@@ -115,7 +147,21 @@ func runScriptCommand() *cli.Command {
 				}
 			}
 
-			fmt.Println("\nScript is running. Please wait for it to finish...")
+			if async {
+				res, err := client.RunHostScriptAsync(h.ID, b, name, c.Uint("team"))
+				if err != nil {
+					if strings.Contains(err.Error(), `Only one of 'script_contents' or 'team_id' is allowed`) {
+						return errors.New("Only one of '--script-path' or '--team' is allowed.")
```

Comment:
very nice detail, we have to figure out a better way to deal with these CLI messages in general, this is not ideal